bot/cogs/snakes.py

```diff
@@ -15,6 +30,62 @@ class Snakes:
     def __init__(self, bot: AutoShardedBot):
         self.bot = bot
 
+    PYTHON_INFO = {
```

Comment:
This probably shouldn't be a class variable, but a module variable declared above class declaration.